docs/controls/risk_assessment/ra-05.md

```diff
@@ -0,0 +1,34 @@
+
+b. Employs vulnerability scanning tools and techniques that facilitate interoperability among tools and automate parts of the vulnerability management process by using standards for:  
+   1. Enumerating platforms, software flaws, and improper configurations;  
+   2. Formatting checklists and test procedures; and  
+   3. Measuring vulnerability impact;  
+
+c. Analyzes vulnerability scan reports and results from security control assessments;  
+
+d. Remediates legitimate vulnerabilities (Assignment: organization-defined response times) in accordance with an organizational assessment of risk; and  
+
+e. Shares information obtained from the vulnerability scanning process and security control assessments with (Assignment: organization-defined personnel or roles) to help eliminate similar vulnerabilities in other information systems (i.e., systemic weaknesses or deficiencies).  
+
+### TDP Implementation
+
+a. As part of the TDP Test Plan, security scans are completed on an ongoing basis, throughout the Continuous Integration (CI).  Automated scans are run on every push, pull request and merge on GitHub.  
+
+b. Security scanning is completed using OWASP ZAP dynamic security scans and Snyk vulnerability dependency scanning.  Snyk will automatically open Pull Requests if there is a vulnerability dependency.  If there are no findings, no Pull Requests will be opened.  OWASP Zap scans are the last step for each CI run.  The results for the scans are summarized and accessed through CircleCI.  
+
+c. Summaries of the security scan reports are reviewed in CircleCI. (see screenshot of summaries of scan reports below)
+
+![screenshot - Summaries of security scan reports](https://github.com/raft-tech/TANF-app/blob/documentation/risk-assessment-ra-5/docs/controls/risk_assessment/ra-5-screenshots/Screen%20Shot%202021-01-29%20at%203.05.42%20PM.png)
+
+d. Summaries of the security scan reports are available in CircleCI.  If there are any vulnerability dependencies found by Snyk, pull requests are automatically opened.  These pull requests are reviewed and remediated as necessary.  
+
+e. Information from the scan reports and control assessments are shared with the appropriate security stakeholders and are available for review in CircleCI. 
+	
+#### Related Files
+[TDP Test Plan Report](https://teams.microsoft.com/l/file/C20CF83B-53DC-45BB-855D-14F513301FE0?tenantId=d58addea-5053-4a80-8499-ba4d944910df&fileType=docx&objectUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA%2FShared%20Documents%2Fcompliance%2FSDLC%2F3_RAR_CDR_TRR%2FTDP_Test%20Plan%20Report%201.21.21.docx&baseUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA&serviceName=teams&threadId=19:3658d5183f3e4e648c0304832c1eafc4@thread.skype&groupId=41f194a6-c1d3-4680-933e-c8ee7d17e287)  
```

Comment:
Folks outside the project team won't have access to these, so I think this can be removed for now and perhaps added back in if/when we have a chance to publish the test plan to our GitHub docs folder.

Reply:
Updated - removed links for now.

Reply:
Realized second link is on GitHub so I think we can keep it